x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/details/components/endpoint_details_tabs.tsx

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useCallback, useMemo, useState } from 'react';
+import styled from 'styled-components';
+import { EuiTabbedContent, EuiTabbedContentTab } from '@elastic/eui';
+
+export enum EndpointDetailsTabsTypes {
+  overview = 'overview',
+  activityLog = 'activity-log',
+}
+
+export type EndpointDetailsTabsId =
+  | EndpointDetailsTabsTypes.overview
+  | EndpointDetailsTabsTypes.activityLog;
+
+interface EndpointDetailsTabs {
+  id: string;
+  name: string;
+  content: JSX.Element;
+}
+
+const StyledEuiTabbedContent = styled(EuiTabbedContent)`
```

Comment:
Can you provide some feedback why we need so much changes to the EUI component?

Reply:
So this bit is copied from the detections where we have tabbed content in a flyout. It would seem it's to ensure the tabs do not scroll along with the content but are fixed below the flyout header. Although, this doesn't work yet with the copied css. I'll clean this up while making sure that tabs are sticky.

Reply:
So I've managed to achieve the behaviour I wanted in 2dd22ed92b3c54da9bdcce467dbe2d7e87097e4d. But after talking to eui team I think we need to use [more-complicated-lyout](https://elastic.github.io/eui/#/layout/flyout#more-complicated-flyout) version of the flyout component to do the tabs nicely. I will have a dig at it in a later commit here.